src/wallet/rpc/spend.cpp

```diff
@@ -758,6 +758,9 @@ RPCHelpMan fundrawtransaction()
                 "in the wallet using importdescriptors (to calculate fees).\n"
                 "You can see whether this is the case by checking the \"solvable\" field in the listunspent output.\n"
                 "Only pay-to-pubkey, multisig, and P2SH versions thereof are currently supported for watch-only\n",
+                "Note that if specifying an exact fee rate, the resulting transaction may have a higher fee rate\n"
+                "if the transaction has unconfirmed inputs, this is because the wallet will attempt to make the\n"
+                "entire package have the given fee rate, not the resulting transaction."
                 {
```

Comment:
```suggestion
                "Only pay-to-pubkey, multisig, and P2SH versions thereof are currently supported for watch-only\n"
                "Note that if specifying an exact fee rate, the resulting transaction may have a higher fee rate\n"
                "if the transaction has unconfirmed inputs, this is because the wallet will attempt to make the\n"
                "entire package have the given fee rate, not the resulting transaction.",
                {
```